assets/search-form.js

```diff
@@ -0,0 +1,20 @@
+class SearchForm extends HTMLElement {
+  constructor() {
+    super();
+    this.input = this.querySelector('input[type="search"]');
+
+    const form = this.querySelector('form.search');
+    form.addEventListener('reset', this.onFormReset.bind(this));
+  }
+
+  onFormReset(event) {
+    // Prevent default so the form reset doesn't set the value gotten from the url on page load
+    event.preventDefault();
+    // Don't reset if the user has selected an element on the predictive search dropdown
+    if (!document.querySelector('a[aria-selected="true"')) {
```

Comment:
I think this is meant to be `li[aria-selected="true"` 🤔 When I check it's the `li` element that is getting the `aria-selected` attribute updated. 

Though I'm not sure the check is working. Or maybe I'm not testing it properly. When I select one item in the dropdown using my keyboard then click on the reset button `X` it still empties the input value. 
[Video recording](https://screenshot.click/22-32-eld40-kvp2w.mp4)